melodies_monet/driver.py

```diff
@@ -937,25 +1000,27 @@ def plotting(self):
                             outname = "{}.{}".format(outname, p_label)
                             # For just the spatial overlay plot, you do not use the model data from the pair file
                             # So get the variable name again since pairing one could be _new.
-                            splots.make_spatial_overlay(
-                                pairdf,
-                                vmodel,
-                                column_o=obsvar,
-                                label_o=p.obs,
-                                column_m=p.model_vars[index],
-                                label_m=p.model,
-                                ylabel=use_ylabel,
-                                vmin=vmin,
-                                vmax=vmax,
-                                nlevels=nlevels,
-                                proj=proj,
-                                outname=outname,
-                                domain_type=domain_type,
-                                domain_name=domain_name,
-                                fig_dict=fig_dict,
-                                text_dict=text_dict,
-                                debug=self.debug
-                            )
+                            # JianHe: only make overplay plots for non-regulatory variables for now
+                            if not cal_reg:
```

Comment:
Can you add a message here like

else:
     print('Warning: Spatial overlay plots are not available yet for regulatory metrics.')